test/scripts/test_private_network.sh

```diff
@@ -35,6 +41,26 @@ ${GOPATH}/bin/goal network stop -r ${NETROOTPATH}
 
 ${GOPATH}/bin/goal network delete -r ${NETROOTPATH}
 
+# Test that genesis generation works correctly
+${GOPATH}/bin/goal network genesis -g ${GENFILESPATH} -t ./test/testdata/nettemplates/TwoNodes50Each.json
+# Try importing genesis files from same template -- should reuse the root and partkey files
+${GOPATH}/bin/goal network create -r ${NETROOTPATH} -n net1 -t ./test/testdata/nettemplates/TwoNodes50Each.json --genesisdir ${GENFILESPATH}
+
+${GOPATH}/bin/goal network start -r ${NETROOTPATH}
```

Comment:
do we need to check the responses from these commend to verify that no error has occurred.

Reply:
We currently don't for any of the private network tests, but we could. Do you have a particular output to check for in mind?

Reply:
I don't have a particular output that I want to check.  from this test, how do we know `${GOPATH}/bin/goal network start -r ${NETROOTPATH}` started the node successfully? should we make a query to check the node status?

Reply:
The `network start` command seem to return errors if there is a problem starting up the network or if the node fails to start: https://github.com/algorand/go-algorand/blob/c797f87766ad5b06e2c2c6a26fe08c6a5e33ed08/cmd/goal/network.go#L209